sandbox/public/index.html

```diff
@@ -42,7 +42,8 @@
         imsOrgId: "53A16ACB5CC1D3760A495C99@AdobeOrg",
         log: true,
         prehidingId: "alloy-prehiding",
-        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }"
+        prehidingStyle: "section:nth-of-type(1) { opacity: 0 !important }",
+        idSyncContainerId: 81
```

Comment:
Is this a required config parameter if idSyncsEnabled?  If so you should add config validation for it.

Reply:
I agree. Even if it isn't required, we should add a config validator to restrict it to a number (assuming it's always a number).

Reply:
It's not required.